### ISSUE TITLE
`loads:` doesn't work for field arguments

### ISSUE THREAD

**Issue by sayduck-daniel** [rmosolgo/graphql-ruby#2544]
I would like to refactor this code:

```ruby
# types/customer_type.rb
module Types
  class CustomerType < BaseCustomObject
    field :groups, Types::GroupType.connection_type, null: false, method: :filter_groups do
      argument :parent_id, String, required: false
    end
  end
end

# models/customer.rb
class Customer
  def filter_groups(parent_id: nil)
    parent = if parent_id.present?
               SayduckSchema.object_from_id(parent_id, nil)
             else
               nil
             end

    groups.where(parent: parent)
  end
end
```

into 

```ruby
# types/customer_type.rb

module Types
  class CustomerType < BaseCustomObject
    field :groups, Types::GroupType.connection_type, null: false, method: :filter_groups do
      argument :parent_id, String, required: false, loads: Types::GroupType
    end
  end
end

# models/customer.rb
class Customer
  def filter_groups(parent: nil)
    groups.where(parent: parent)
  end
end
```

The documentation mentions this `loads:` attribute but it is not behaving correctly. 

With this sample code: 
```ruby
class Customer
  def filter_groups(parent: nil, parent_id: nil)
    pp "parent: #{parent}"
    pp "parent: #{parent.class}"
    pp "parent_id: #{parent_id}"
    groups.where(parent: parent)
  end
end
```

I get this in my log:
```log
Started POST "/graphql" for 127.0.0.1 at 2019-10-14 13:18:10 +0300
Processing by GraphqlController#execute as HTML
  Parameters: ...
  User Load (0.3ms)  ...
  ↳ app/controllers/concerns/authenticate.rb:27
  Customer Load (0.3ms)  ...
  ↳ app/graphql/sayduck_schema.rb:27
"parent: R3JvdXAtNGE1YjNlZTAtZDA5OS0wMTM3LWM4YWItMGM4NTkwOWQ4YmYw"
"parent: String"
"parent_id: "
  Group Load (0.6ms) ...
  ↳ app/graphql/sayduck_schema.rb:27
Completed 200 OK in 2249ms (Views: 0.2ms | ActiveRecord: 14.8ms)
```

Not how the `parent` is simply a string and how no queries for group have been made before the debug code. It should at least have made a find_by for the parent group.

Any idea what could be wrong here?

**Comment by daemonsy** [comment]
Hi @sayduck-daniel do you have `.object_from_id` defined on `SayduckSchema`? 

https://graphql-ruby.org/mutations/mutation_classes.html#auto-loading-arguments

**Comment by sayduck-daniel** [comment]
Yes @daemonsy, `.object_from_id` works. On my first code you can see the working alternative using it:

```ruby
# types/customer_type.rb
module Types
  class CustomerType < BaseCustomObject
    field :groups, Types::GroupType.connection_type, null: false, method: :filter_groups do
      argument :parent_id, String, required: false
    end
  end
end

# models/customer.rb
class Customer
  def filter_groups(parent_id: nil)
    parent = if parent_id.present?
               SayduckSchema.object_from_id(parent_id, nil) # <- here
             else
               nil
             end

    groups.where(parent: parent)
  end
end
```

**Comment by rmosolgo** [comment]
Sorry @sayduck-daniel , `loads:` is supported in resolvers and input objects, but not on plain field arguments. There's no _reason_ for this, except that the first place I wanted `loads:` was mutations, but it's been slowly added to other places. 

For now, you could refactor the field to use a resolver if you want to use `loads:`. 

Let's keep this open to track `loads:` with plain field arguments; I'll try to add it in 1.10 or someone else can add it sooner if they want.

**Comment by rmosolgo** [comment]
Fixed by #2720
